department.rb

```diff
@@ -0,0 +1,55 @@
+class Department
+  attr_reader :department_name, :department_raise, :total_salary
+  def initialize(department_name)
+    @department_name=department_name
+    @employees=[]
+
+  end
+
+  def add(employee)
+    employee.department = @department_name
```

Comment:
Nice to keep a variable on employee that points in the opposite direction.  However, why not do `employee.department = self`?  That way you'll have access to the whole department.